src/components/browse/singleDatasetView/singleDatasetViewTabs/singleDatasetViewTabs.tsx

```diff
@@ -24,23 +24,27 @@ const SingleDatasetViewTabs = (props: SingleDatasetViewTabsProps) => {
           <Nav.Item>
             <Nav.Link
               eventKey="0"
-              className="border border-1 mx-2 border-light"
+              className="border border-1 mx-2 border-light text-center"
+              style={{width: "150px", borderRadius:"10px"}}
             >
               <FontAwesomeIcon
-                icon={faChartPie}
-                className="text-secondary me-2"
+                icon={faBook}
+                className="text-secondary me-2 rounded"
+                style={{width: "15px", height: "15px", backgroundColor: "rgba(214,95,48,0.2)", padding:"4px"}}
               />
               Study
             </Nav.Link>
           </Nav.Item>
           <Nav.Item>
             <Nav.Link
               eventKey="1"
-              className="border border-1 mx-2 border-light"
+              className="border border-1 mx-2 border-light text-center"
```

Comment:
Can you align text and icons in the same line?

Reply:
not done

Reply:
Fixed